types/globaloffensive/enums.d.ts

```diff
@@ -0,0 +1,28 @@
+export enum GCConnectionStatus {
+    HAVE_SESSION = 0,
+	GC_GOING_DOWN = 1,
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/globaloffensive/enums.d.ts;
  Line: 3;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.